server/utils/statusTag.ts

```diff
@@ -103,3 +102,34 @@ export class TaskStatusTag extends StatusTag<TaskStatus> {
     })
   }
 }
+
+export class BookingStatusTag extends StatusTag<BookingStatus> {
```

Comment:
I wonder if it might be better to have these in seperate files namespaced into the appropriate folder, i.e `utils/bookings/bookingStatus.ts` etc etc

Reply:
yeah good shout